src/axom/bump/TopologyMapper.hpp

```diff
@@ -690,7 +690,11 @@ class TopologyMapper
 #endif
                 matids[m] = mat;
                 vfs[m] = vf;
-                sizes[zi]++;
+                // Workaround for failure on rzvernal (misreporting out-of-bounds access)
+                if(zi < sizes.size())
+                {
+                  sizes[zi]++;
+                }
```

Comment:
I added this for a segfault failure in `bump_topology_mapper` unit test I was seeing on rzvernal. Not entirely sure what the actual cause is. Printing out the indices don't reveal it to be out-of-order, also doesn't fail when a print statement is added, maybe a race condition?

Reply:
Issue added #1705 !